PWGCF/FEMTOSCOPY/AliFemtoUser/PWGCFfemtoscopyUserLinkDef.h

```diff
@@ -126,6 +126,14 @@
 #pragma link C++ class AliFemtoConfigObject-;
 #pragma link C++ class AliFemtoConfigObject::Painter;
 #pragma link C++ class AliFemtoConfigObject::BuildMap;
+#pragma link C++ class AliFemtoConfigObject::list_iterator;
+#pragma link C++ class AliFemtoConfigObject::map_iterator;
+#pragma link C++ class std::map<std::string, AliFemtoConfigObject>;
+#pragma link C++ class std::vector<AliFemtoConfigObject>;
+#pragma link C++ class std::vector<AliFemtoConfigObject>::iterator;
+#pragma link C++ class std::pair<double, double>;
```

Comment:
This is causing a warning with the root maps in root 6. Specifically, `Warning in <TInterpreter::ReadRootmapFile>: class  pair<double,double> found in libCore.so  is already in libPWGCFfemtoscopyUser`. My understanding is that at least some of these more common link defs are already taken care of in root 5 as well (although I haven't confirmed). If possible, can you please remove this entry?

Reply:
Thanks and yes, I'm aware; it's been removed in my local branch but I was going to finish a new feature before merging it back into main.